<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>821</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>791</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Board ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="boardID">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Axis</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Axis">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Adobe Heiti Std</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Target Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="targetPosition">
            <property name="font">
             <font>
              <family>Adobe Heiti Std</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-50000</number>
            </property>
            <property name="maximum">
             <number>50000</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Adobe Heiti Std</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Current Position </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="current_position">
                <property name="font">
                 <font>
                  <family>Adobe Heiti Std</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="acceptDrops">
                 <bool>false</bool>
                </property>
                <property name="value" stdset="0">
                 <double>0.000000000000000</double>
                </property>
                <property name="intValue" stdset="0">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Velocity(RPM)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="velecity">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Acceleration(RPS/S)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="acceletation">
          <property name="font">
           <font>
            <family>Adobe Heiti Std</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Adobe Heiti Std</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Deceleration(RPS/S)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="deceleration">
            <property name="font">
             <font>
              <family>Adobe Heiti Std</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>180</y>
       <width>156</width>
       <height>23</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50000</number>
     </property>
     <property name="maximum">
      <number>50000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
